http-clients/src/main/java/com/palantir/remoting/http/FeignClientFactory.java

```diff
@@ -134,6 +139,41 @@ public static FeignClientFactory of(
     }
 
     /**
+     * Constructs a dynamic proxy for the specified type using the supplied {@link ServiceConfiguration}. If the SSL
+     * configuration is not present in the {@link ServiceConfiguration}, the fallback SSL configuration will be used.
+     */
+    public <T> T createProxies(
+            ServiceConfiguration serviceConfig,
+            Optional<SslConfiguration> fallbackSslConfiguration,
```

Comment:
there is no defaut config for ServiceConfiguration. Default config is from ServiceDiscoveryConfiguration